packages/cli/src/commands/deviceprofiles/view.ts

```diff
@@ -1,8 +1,6 @@
-import { DeviceProfile } from '@smartthings/core-sdk'
+import { APIOrganizationCommand, formatAndWriteItem, outputItemOrList } from '@smartthings/cli-lib'
```

Comment:
Similar to `presentation.ts`, we shouldn't need `outputItemOrList` here anymore and the flags should come from `formatAndWriteItem`.

Reply:
Done.